deploy/manifests/quay-operator/3.4/quay-operator.clusterserviceversion.yaml

```diff
@@ -2,11 +2,11 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
-    olm.skipRange: ">=3.3.x <3.4.0"
+    olm.skipRange: ">=3.3.x <3.4.3"
     capabilities: Full Lifecycle
     categories: Integration & Delivery
-    containerImage: registry-proxy.engineering.redhat.com/rh-osbs/quay-quay-quay-operator-container:v3.4.0
-    createdAt: 2021-01-05 16:01 UTC
+    containerImage: registry-proxy.engineering.redhat.com/rh-osbs/quay-quay-operator-rhel8:v3.4.3
+    createdAt: 2021-03-16 15:03 UTC
     description: Opinionated deployment of Red Hat Quay on Kubernetes.
```

Comment:
Could you add the following annotations:

```yaml
quay-version: 3.4.0
quay-upgradeable-versions: '["vader"]'
```

We want `quay-upgradeable-versions` to be JSON-in-YAML.

Reply:
let's coordinate in the other PR on these files